docs/en/table_design/table_types/primary_key_table.md

```diff
@@ -1,159 +1,212 @@
+
+If the disk is a SSD, it is recommended to set it to true. If the disk is HDD and the load frequency is not high, you can also set it to true.
+
+:::note
 
-## Usage notes
+Since v3.1.4, Primary Key tables created in StarRocks shared-data clusters further support index persistence onto local disks.
 
-- Take note of the following points about the primary key of a table:
-  - The primary key is defined by using the `PRIMARY KEY` keyword.
+:::
 
-  - The primary key must be created on columns on which unique constraints are enforced, and the names of the primary key columns cannot be changed.
+</TabItem>
 
-  - The primary key columns can be any of the following data types: BOOLEAN, TINYINT, SMALLINT, INT, BIGINT, LARGEINT, STRING, VARCHAR, DATE, and DATETIME. However, the primary key columns cannot be defined as `NULL`.
+<TabItem value="example2" label="fully in-memory primary key index">
 
-  - The partition column and the bucket column must participate in the primary key.
+When `enable_persistent_index` is set to `false`, the primary key indexes are not persisted to the disk,  that is, the primary key indexes are fully stored in memory. During loading, the primary key indexes of tablets related to the data loaded will be loaded into memory, which may result in higher memory consumption. (If a tablet has not had data loaded for a long time, its primary key index will be released from memory).
 
-  - The number and total length of primary key columns must be properly designed to save memory. We recommend that you identify columns whose data types occupy less memory and define those columns as the primary key. Such data types include INT and BIGINT. We recommend that you do not let a column of the VARCHAR data type to participate in the primary key.
+When using the fully in-memory primary key indexes, it is recommended to follow the following guidelines when designing the Primary Key to control memory usage of the primary key indexes:
 
-  - Before you create the table, we recommend that you estimate the memory occupied by the primary key index based on the data types of the primary key columns and the number of rows in the table. This way, you can prevent the table from running out of memory. The following example explains how to calculate the memory occupied by the primary key index:
-    - Suppose that the `dt` column, which is of the DATE data type that occupies 4 bytes, and the `id` column, which is of the BIGINT data type that occupies 8 bytes, are defined as the primary key. In this case, the primary key is 12 bytes in length.
+- The number and total length of Primary Key columns must be properly designed. We recommend that you identify columns whose data types occupy less memory and define those columns as the Primary Key, such as INT and BIGINT, not VARCHAR.
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/a4c9d86c-3e4b-47f2-9dbf-9afdfc663e28)
-> primary key
Check the entire doc and fix the letter case for "Primary Key". Note that we say "the primary key" rather than "the Primary Key", as this letter case is more popular.